enrollment-server-domain-model/src/main/java/com/wultra/app/enrollmentserver/model/enumeration/DocumentStatus.java

```diff
@@ -65,12 +65,21 @@ public enum DocumentStatus {
     FAILED;
 
     /**
-     * All not finished statuses
+     * All not finished statuses.
      */
     public static final List<DocumentStatus> ALL_NOT_FINISHED = ImmutableList.of(
             UPLOAD_IN_PROGRESS,
             VERIFICATION_PENDING,
             VERIFICATION_IN_PROGRESS
     );
 
+    /**
+     * All unsuccessful statuses.
+     */
+    public static final List<DocumentStatus> ALL_FAILED = ImmutableList.of(
```

Comment:
Have you considered to introduce a property to the enum and filter values based on it? I mean, it is more error proof when we add another enum values not to forget edit this list.

Reply:
We can improve it in a separate issue, if you feel strongly about it. I cannot think of any other states right now.

Reply:
I can live with that so far ;) I just wanted to mention a way, how it is possible to work with enums next time.